### PR TITLE
Rename pharos to velum

### PR DESCRIPTION
**Pull request by jimmykarily**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,38 +8,38 @@
...
 After any change that you have performed you can run any rails/rake task as if it was local, with
-the docker prefix, like `docker exec -it $(docker ps | grep pharos-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rspec"` or `docker exec -it $(docker ps | grep pharos-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rubocop"`.
+the docker prefix, like `docker exec -it $(docker ps | grep velum-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rspec"` or `docker exec -it $(docker ps | grep velum-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rubocop"`.
 
 ## Licensing
 
-Pharos is licensed under the Apache License, Version 2.0. See
-[LICENSE](https://github.com/SUSE/Pharos/blob/master/LICENSE) for the full
+Velum is licensed under the Apache License, Version 2.0. See
+[LICENSE](https://github.com/kubic-project/Velum/blob/master/LICENSE) for the full
```
Comment (MaximilianMeister) [inline README.md]: should work too, but just to be correct i'd change the link to small case `s/Velum/velum`

**Comment by ereslibre** [comment]
LGTM :+1:

File: README.md (hunk 1 above)
Comment (jimmykarily) [inline README.md]: fixed

**Comment by MaximilianMeister** [comment]
I just pushed the new image to dockerhub

**Pull request merged**